addons/web/static/tests/helpers/qunit_config.js

```diff
@@ -155,6 +158,17 @@ QUnit.moduleDone(function(result) {
 
 });
 
+const callbacks = [];
+QUnit.addCleanup = function (cb) {
```

Comment:
run fact: we have in the master-wowl branch a `registerCleanup` function that look really close to this one: https://github.com/odoo-dev/odoo/blob/master-wowl/addons/web/static/tests/helpers/cleanup.js#L18

Reply:
Hooked on `OdooAfterTestHook` but only allowing synchronous cleanup though.

I'd hooked mine on `testDone` so it could be asynchronous, sadly that didn't really work out because of the `OdooAfterTestHook` running *earlier*.

Ultimately I removed the one I had because I wanted to cleanup stuff `OdooAfterTestHook` tests for, but I also needed an asynchronous cleanup.